shell/app/common/components/contractive-filter.tsx

```diff
@@ -783,7 +783,7 @@ export const ContractiveFilter = ({
             }
             placement="bottomLeft"
           >
-            <span className="contractive-filter-item">
+            <span className="contractive-filter-item more-conditions">
```

Comment:
the className `more-conditions`  is only for test code?

Reply:
yes